js/src/state-pages.js

```diff
@@ -0,0 +1,17 @@
+(function(exports) {
+  'use strict';
+
+  require('./../components/bar-chart-table.js');
+  require('./../components/data-map.js');
+
+  require('./../components/aria-tabs.js');
+  require('./../components/sticky-nav.js');
+  exports.OpenListNav = require('./../components/open-list-nav.js');
+
+  // exporting instance of OpenListNav because openListNav is
+  // referenced in the markup:
+  // _includes/hash_selecor.html
+  exports.openListNav = new OpenListNav();
```

Comment:
[It definitely is defined](https://github.com/18F/doi-extractives-data/pull/1449/files#diff-fbca2654d407c04806d7604120796e80R10)